src/index.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.scss';
 import { Provider } from 'react-redux';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
I think you want the SCSS files, not the CSS.

Reply:
it defaults to css actually. At least for the npm package npm install bootstrap. 
I did see a npm package for bootstrap scss... but it didn't look as reliable. Unless I'm missing something on my side. Just didn't want to download any weird packages.